rclpy/src/rclpy/_rclpy.c

```diff
@@ -3613,7 +3994,7 @@ static PyMethodDef rclpy_methods[] = {
 
   {
     "rclpy_convert_from_py_qos_policy", rclpy_convert_from_py_qos_policy, METH_VARARGS,
-    "Convert a QoSPolicy python object into a rmw_qos_profile_t."
+    "Convert a QoSPolicy Python object into a rmw_qos_profile_t."
```

Comment:
@mikaelarguedas had me turn all my `python` references to `Python` so I brought this one along for the ride. Has nothing else to do with this PR.